packages/beacon-node/src/api/impl/validator/index.ts

```diff
@@ -335,6 +332,26 @@ export function getValidatorApi({
       );
   }
 
+  function getDefaultGraffiti(): string {
```

Comment:
Looking at code in this PR, it seems a bit scattered around, we should consider moving this into utils file and having graffiti related code there, like `ClientVersion`. Would also allow to add unit tests for this which would document the expected format as well.

All graffiti related codes, utils and types should be in a graffiti.ts (or similar) file. The execution engine is responsible for providing the version of the connected EL and that should be it.